.github/workflows/check-for-build-warnings.yml

```diff
@@ -0,0 +1,10 @@
+on: [pull_request]
+
+jobs:
+  status_checker_job:
+    runs-on: ubuntu-latest
+    name: Checks the OPS status check for build warnings
+    steps:
+    - uses: dotnet/samples/.github/actions/status-checker@main
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we want it to run on all branches?
For example, see https://github.com/dotnet/docs/pull/23568 which doesn't target `main` or `live`, do we want to run this check in such PRs?

Reply:
I think so. Let's see how much noise it generates.

Reply:
Yes, I'd say we want this to run on all branches.